portfolio/src/main/webapp/index.html

```diff
@@ -5,15 +5,33 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
+        body {
+            background-image: url('https://i.gifer.com/GHSe.gif');
+            }
+    </style>
+    <link href="https://fonts.googleapis.com/css?family=Annie+Use+Your+Telescope|Nanum+Pen+Script&display=swap" rel="stylesheet">
   </head>
   <body>
+      <div class="nav">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">News</a>
+  <a href="#contact">Contact</a>
+  <a href="about.html">About Me</a>
+  </div>
```

Comment:
Minor style thing here, try to keep the indentation consistent and indent nested elements more than the parent. Our style guide recommends 2 spaces, https://google.github.io/styleguide/htmlcssguide.html#Indentation.

Reply:
Got it! I haven't gotten around to fixing the small details yet, but i will keep a look out for that.